runtime/onert/frontend/trix/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+if (NOT BUILD_TRIX_LOADER)
+  return()
+endif ()
+
+set(SOURCES src/trix_loader_base.cc)
+
+nnfw_find_package(TRIXEngine EXACT 2.5.0 QUIET)
+if(TRIXEngine_FOUND)
+  message(WARNING "TRIXEngine found")
+  list(APPEND SOURCES src/trix_loader.cc)
+else()
+  message(WARNING "TRIXEngine NOT found, trix loader is dummy.")
+  list(APPEND SOURCES src/trix_loader_dummy.cc)
```

Comment:
If TrixEngine is not available, dummy trix loader will be used. It has no dependency to trix_engine. Literally, it is a dummy. : )